src/plugins/dam/components/search-assets-screen/index.js

```diff
@@ -98,10 +100,10 @@ function SearchAssetsScreen(props) {
                 const transcodingStatus = node.get('transcoding_status');
 
                 return (
-                  <tr key={`${node.get('_id')}`} className={`status-${node.get('status')}`}>
+                  <tr key={`${node.get('_id')}`} className={`status-${node.get('status')}`} style={{cursor: 'pointer'}}>
```

Comment:
@rsumilang use the `cursor-pointer` utility class in className instead.

Reply:
@gdbrown there seems to be a bug in chrome when using the class name instead. the `cursor: pointer` setting is inherited down to each `td` but the cursor does not change. only changes when using the class if applied to each individual `td`.